support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -112,7 +113,25 @@ class MarketingConsentWithCheckbox extends Component<PropTypes, StateTypes> {
   }
 
   render() {
-    const emailFrequency = this.props.countryGroupId === 'AUDCountries' ? 'an occasional' : 'a weekly';
+    const createTitleText = () => {
```

Comment:
Can this be a function outside of the render loop? I think it's desirable/more readable to keep functions outside of methods inside a class. You can pass it the relevant props when it's called. Ping me if that needs clarifying?

Reply:
👍